### PR TITLE
update http quickstart with working api urls

### LINKED ISSUE #316: HTTP Quickstart is currently broken as the paths are out of date
https://www.tigrisdata.com/docs/quickstarts/quickstart-http/

### PR DESCRIPTION
**Pull request by ibrahim-akrab**
This PR closes #316 

I found the search section not applicable without an index so I got rid of it. An alternative would have been creating the index but it is the same steps as creating a collection and it seemed very repetitive. I can add the necessary steps to create the index and search if needed.

/claim #316

### CODE REVIEW

File: docs/quickstarts/quickstart-http.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ Tigris, a collection is an ordered set of records called documents that all
 follow a specific schema and whose fields are strongly typed.
 
 ```shell
-curl https://api.preview.tigrisdata.cloud/v1/databases/sampledb/collections/users/createOrUpdate \
+curl https://api.preview.tigrisdata.cloud/v1/projects/quickstart/database/collections/users/createOrUpdate \
```
Comment (ovaistariq) [inline docs/quickstarts/quickstart-http.mdx:64]: Could you update the schema below to demonstrate making `name` searchable?

```
"balance": {
              "type": "number",
              "searchIndex": true
            },
"name": {
              "type": "string",
              "searchIndex": true
            }
```

Comment (ovaistariq) [reply docs/quickstarts/quickstart-http.mdx:64]: Then once you have done that, you can add an API example for making a search request on the `name` field.

**Comment by ibrahim-akrab** [comment]
I'll prettify it and try again

**Comment by ibrahim-akrab** [comment]
I'll prettify it and try again

File: docs/quickstarts/quickstart-http.mdx [hunk 2]
```diff
@@ -158,29 +158,13 @@ curl https://api.preview.tigrisdata.cloud/v1/databases/sampledb/collections/user
       }'
 ```
 
-### Search
```
Comment (ovaistariq) [inline docs/quickstarts/quickstart-http.mdx:161]: Check my comment above. Once you have updated the schema to mark fields as searchable, this API request will start to work.

**Review by ovaistariq — CHANGES_REQUESTED** [review]

File: docs/quickstarts/quickstart-http.mdx (hunk 1 above)
Comment (ibrahim-akrab) [reply docs/quickstarts/quickstart-http.mdx:64]: Sure thing. Will update it and check the full quickstart.

**Comment by ibrahim-akrab** [comment]
@ovaistariq It should be all good now.

I'd like to mention something I noticed while updating this quickstart. The application key created by the CLI registers on the web interface, however, it doesn't generate an access token reliably. I've tried creating the key using the web interface then generating the access token using the CLI and it works flawlessly.

**Comment by ovaistariq** [comment]
> @ovaistariq It should be all good now.
> 
> I'd like to mention something I noticed while updating this quickstart. The application key created by the CLI registers on the web interface, however, it doesn't generate an access token reliably. I've tried creating the key using the web interface then generating the access token using the CLI and it works flawlessly. When the CLI fails, it returns `{"error":{"code":"UNAUTHENTICATED","message":"Invalid credentials"}}`

Thanks for letting me know about this. So if I understand correctly, if the application key is created using CLI, then the token mechanism doesn't work reliably. But if the application key is created using the web console then everything work as expected?

**Review by ovaistariq — APPROVED** [review]

**Comment by ibrahim-akrab** [comment]
@ovaistariq Yes, that's what happens.

**Comment by TheLazyIndianTechie** [comment]
Yup, I have the same issue. Let me try the fix @ibrahim-akrab has mentioned.

**Comment by ovaistariq** [comment]
There were a couple more fixes that I have done on the HTTP quickstart. Let me know if you still run into issues: https://www.tigrisdata.com/docs/quickstarts/quickstart-http/

**Pull request merged**